config/peerpods/podvm/lib.sh

```diff
@@ -196,11 +196,14 @@ function prepare_source_code() {
             error_exit "Failed to enable fips mode"
     fi
 
+    # links must be relative
     if [[ "$CONFIDENTIAL_COMPUTE_ENABLED" == "yes" ]]; then
-        sed 's/default SetPolicyRequest := true/default SetPolicyRequest := false/; s/default ExecProcessRequest := true/default ExecProcessRequest := false/' \
+        sed 's/default ReadStreamRequest := true/default ReadStreamRequest := false/;
+            s/default ExecProcessRequest := true/default ExecProcessRequest := false/' \
             "${podvm_dir}"/files/etc/kata-opa/default-policy.rego > "${podvm_dir}"/files/etc/kata-opa/coco-default-policy.rego
-        ln -sf "${podvm_dir}"/files/etc/kata-opa/coco-default-policy.rego  "${podvm_dir}"/files/etc/kata-opa/default-policy.rego
+        ln -sf coco-default-policy.rego "${podvm_dir}"/files/etc/kata-opa/default-policy.rego
```

Comment:
Hmm is the above correct?
the coco-default-policy.rego is in `"${podvm_dir}"/files/etc/kata-opa/`

Reply:
> Hmm is the above correct? the coco-default-policy.rego is in `"${podvm_dir}"/files/etc/kata-opa/`

Yes it is because they both files are in the same directory. When the VFS will try to access the right most component of `"${podvm_dir}"/files/etc/kata-opa/default-policy.rego`  from `"${podvm_dir}"/files/etc/kata-opa/`, it will go to the exact content of the symlink, i.e. `coco-default-policy.rego` which exists in the directory.

The first argument to `ln -s` is basically arbitrary text that the VFS is expected to use when resolving the symlink. The `symlink(2)` syscall doesn't even require the target to exist.